pkg/asset/installconfig/stock.go

```diff
@@ -51,12 +51,14 @@ func (s *StockImpl) EstablishStock(directory string, inputReader *bufio.Reader)
 	}
 	s.clusterID = &clusterID{}
 	s.emailAddress = &asset.UserProvided{
-		Prompt:      "Email Address:",
-		InputReader: inputReader,
+		Prompt:       "Email Address:",
+		DefaultValue: "a@b.c",
```

Comment:
This is a pretty terrible default ;).  If we would actually accept this value, I'd rather just make the field `omitempty` and optional.